components/GraphDiagram/GraphDiagram.scss

```diff
@@ -95,20 +96,24 @@ g.node {
 
   &.selected {
     & > path {
-      stroke-width: 3;
-      stroke: var(--highlight) !important;
+      fill: var(--accent) !important;
+      stroke: var(--accent) !important;
+      stroke-width: 5;
+    }
+    & > text {
+      fill: var(--text-on-accent);
     }
   }
 
   &.upstream {
     & > path {
-      stroke: var(--stroke-blue);
+      stroke: var(--accent-100);
     }
   }
 
   &.downstream {
     & > path {
-      stroke: var(--stroke-violet);
+      stroke: var(--accent);
```

Comment:
Dependencies: they're still part of the selected module, so I believe they should have the same color. 

The selected module is now fully orange so it's still visually different.

Reply:
Yeah, I agree having the dependency colors closer to, if not the same, as the selection color makes sense.